MQ2ItemScore.cpp

```diff
@@ -806,6 +807,7 @@ int SetClickMode(int Val, char* Opt)
 void SetClickMode(char* Val, char* Opt)
 {
 	if (_stricmp(Val, "Channel") == 0) strcpy_s(ClickChannel, Opt);
+	// TODO: Check the Validity of the !Clicky___
```

Comment:
what validity do you want to check? These look like a toggle.

Reply:
Just that it is valid usage/intent. Visual studio says ~ was probably intended, I'm guessing just "0" was intended, but wanted to come back to that

Reply:
Apart from the fact that these variables should've been written as `bool`, theres nothing inherently wrong with the logic here.